src/conversion.jl

```diff
@@ -37,12 +38,18 @@ function generatorproducer(m::RepMatrix)
         end
     end
 end
+
 function enumtomat(m::RepMatrix)
-    M = Matrix{Rational{BigInt}}(undef, 0, fulldim(m)+1)
-    for output in generatorproducer(m)
-        M = [M; output']
+    rows = Vector{Vector{Rational{BigInt}}}(undef, 0)
+    sizehint!(rows, fulldim(m)) # heuristic size hint
```

Comment:
Why is the dimension a good estimation of the number of vertices ?

Reply:
Just sounded fine, I just need a good-enough estimation to avoid re-allocating the vector all the time

Reply:
If I'm not mistaken, dynamic vectors double the inner capacity every time you hit the max cap

Reply:
Yes, so you want to avoid the reallocation for the first few powers of two ?

Reply:
yes, and the number of dimensions seems like a reasonable almost lower bound for the number of vertices to expect